### ISSUE TITLE
Logging: flask-ldap3-login adds the console handler to the root level logger

### ISSUE THREAD

**Issue by nsmcan** [nickw444/flask-ldap3-login#107]
**flask-ldap3-login** inadvertently adds the console handler to the root level logger by calling an uninitialized handler in **forms.py**:
```
logging.debug('Validating LDAPLoginForm against LDAP')
...
logging.debug("Form validation failed before we had a chance to check ldap. Reasons: '{0}'".format(self.errors))
```

This causes log messages from all other, unrelated, loggers to be duplicated on the console.

Could you please kindly fix code by using the **log** variable instead, which you already defined in code on the line 8

```
log.debug('Validating LDAPLoginForm against LDAP')
...
log.debug("Form validation failed before we had a chance to check ldap. Reasons: '{0}'".format(self.errors))
```

**Comment by gmacon** [comment]
I'd happily merge a PR with this change if someone wants to write it.

**Comment by nsmcan** [comment]
I'll prepare it soon

**Comment by nickw444** [comment]
Fixed by #108
